src/_gettsim_tests/test_synthetic.py

```diff
@@ -1,50 +1,120 @@
 from __future__ import annotations
 
-from _gettsim.config import numpy_or_jax as np
-from _gettsim.interface import compute_taxes_and_transfers
+import pandas as pd
+import pytest
 from _gettsim.synthetic import create_synthetic_data
 
-from _gettsim_tests._helpers import cached_set_up_policy_environment
 
-
-def test_synthetic():
-    """Test creation of synthetic data."""
-    # run with defaults
+def test_default():
```

Comment:
```suggestion
def test_with_defaults():
```

Otherwise I expect the default to be tested not to change or so; these are rather sanity checks.